server/controllers/error_handling_controllers.js

```diff
@@ -22,6 +22,12 @@ exports.handlePSQL400 = (err, req, res, next) => {
 exports.handleCustomErrors = (err, req, res, next) => {
   if (err === "No article of that id found") {
     res.status(404).send({ msg: err });
+  } else if (err === "That topic doesn't exist") {
+    res.status(404).send({ msg: err });
+  } else if (err === "That column doesn't exist") {
+    res.status(404).send({ msg: err });
+  } else if (err === "Invalid order") {
+    res.status(400).send({ msg: err });
```

Comment:
This handling is okay but could be cleaner if you just pass an error object into your Promise.rejects. 

Something like `Promise.reject({status:400, msg: 'bad request'})

Then your error handler has all it need to send the response and you won't need the `if else if` logic.